### PR TITLE
Added labels to interactive visualiser (finos#568)

### PR DESCRIPTION
**Pull request by oliviajanejohns**
See screenshot below:
Screenshot 2024-11-07 at 14 39 00

I've looked further into JSPlump and it seems that it doesn't provide an easy way of preventing overrides of labels out of the box. Will improve in a future PR but also would suggest utilising a different graph library for example D3

### CODE REVIEW

File: calm-visualizer/src/components/graph/Graph.tsx [hunk 1]
```diff
@@ -29,7 +27,8 @@ function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationsh
             options: {}
         },
         overlays: [
-            { type:"Arrow", options:{location:1}}
+            { type:"Arrow", options:{location:1}},
+            {  type: "Label", options: { location: 0.5, label: relationship.description }}
```
Comment (willosborne) [inline calm-visualizer/src/components/graph/Graph.tsx:31]: Might want to make the formatting a bit more consistent here

Comment (willosborne) [inline calm-visualizer/src/components/graph/Graph.tsx:31]: Will this cause any problems if there isn't a description defined on the relationship?

**Comment by willosborne** [comment]
Looks cool and adds a lot of detail. 
Some of the lines of text are very long and quite hard to follow; is there any way we could wrap (or even truncate with an on-hover expansion) text that goes over a certain size?

**Review by willosborne — CHANGES_REQUESTED** [review]

File: calm-visualizer/src/components/graph/Graph.tsx (hunk 1 above)
Comment (oliviajanejohns) [reply calm-visualizer/src/components/graph/Graph.tsx:31]: label also accepts undefined as a value in the case the description isnt defined

**Comment by oliviajanejohns** [comment]
Adjusted styling slightly - however looking at perhaps using d3.js or other libraries:
Screenshot 2024-11-12 at 00 24 38

**Review by aidanm3341 — APPROVED** [review]
